web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -133,6 +137,32 @@ public boolean include(Table table) {
         };
     }
 
+    @GET
+    @Path ("_changedtables")
+    @Timed (name = "bv.emodb.sor.DataStoreResource1.listMetadataChangedTables", absolute = true)
+    @Unbuffered
+    @ApiOperation (value = "Returns all the emo tables whose metadata was changed.",
+            notes = "Returns a Iterator of a Map.",
+            response = Table.class
+    )
+    public Object listMetadataChangedTables(final @QueryParam ("from") DateTimeParam fromInclusiveParam,
+                                            final @QueryParam ("to") DateTimeParam toExclusiveParam,
+                                            final @QueryParam ("limit") @DefaultValue ("10") LongParam limitParam,
```

Comment:
Also mentioned earlier, there's no benefit of a "limit" if there's no way to paginate if more than "limit" records are returned.